arborist/resource.go

```diff
@@ -103,7 +103,8 @@ func NewResource(
 		}
 		pathSegments[len(parent.pathSegments)] = name
 	} else {
-		pathSegments = []string{name}
+		//pathSegments = []string{name}
```

Comment:
delete the comment line

Reply:
Fixed in bc25eb4ad8ccd44d86b7a24694e3989552e789c7